ingeniamotion/fsoe_master/handler.py

```diff
@@ -186,6 +188,29 @@ def __init__(
             self._master_handler.delete()
             raise ex
 
+        # SCI serializer
+        self._sci_serializer = SCISerializer()
```

Comment:
Does the serializer have a state that we have to retain? Or we can create it on every serialize_mapping_to_sci call?